react/src/components/ServiceLauncherPageContent.tsx

```diff
@@ -256,7 +257,7 @@ const ServiceLauncherPageContent: React.FC<ServiceLauncherPageContentProps> = ({
     return {
       image: (isManualImageEnabled
         ? formInput.environments.manual?.split('@')[0]
-        : `${formInput.environments.image?.registry}/${formInput.environments.image?.name}:${formInput.environments.image?.tag}`) as string,
+        : `${formInput.environments.image?.registry}/${formInput.environments.image?.namespace ?? formInput.environments.image?.name}:${formInput.environments.image?.tag}`) as string,
```

Comment:
It's the same as below. We don't need to generate the full image name. It already exists in ResourceAllocationFormItems.
```suggestion
        : formInput.environments.version.split('@')[0]) as string,
```